docs/en/advanced_guides/runtime.md

```diff
@@ -0,0 +1,261 @@
+
+## Load Checkpoint / Resume Training
+
+In config files, you can specify the loading and resuming functionality as below:
+
+```python
+# load from which checkpoint
+load_from = "Your checkpoint path"
+
+# whether to resume training from the loaded checkpoint
+resume = False
+```
+
+The `load_from` field can be either a local path or an HTTP path. And you can resume training from the checkpoint by
+speicfy `resume=True`.
+
+```{tip}
+You can also enable auto resuming from the latest checkpoint by specifying `load_from=None` and `resume=True`.
+```
+
+If you are training models by our `tools/train.py` script, you can also use `--resume` argument to resume
+training without modifying the config file manually.
+
+```bash
+# Automatically resume from the latest checkpoint.
+python tools/train.py configs/resnet/resnet50_8xb32_in1k.py --resume
+
+# Resume from the specified checkpoint.
+python tools/train.py configs/resnet/resnet50_8xb32_in1k.py --resume checkpoints/resnet.pth
+```
+
+## Randomness Configuration
+
+In the `randomness` field, we provide some options to make the experiment as reproducible as possible.
+
+By default, we won't specify seed in the config file, and in every experiment, the program will generate a random seed.
+
+**Default settings:**
+
+```python
+randomness = dict(seed=None, deterministic=False)
+```
+
+To make the experiment more reproducible, you can specify a seed and set `deterministic=True`. The influence
+of the `deterministic` option can be found [here](https://pytorch.org/docs/stable/notes/randomness.html#cuda-convolution-benchmarking).
+
+## Log Configuration
+
+The log configuration relates to multiple fields.
+
+In the `log_level` field, you can specify the global logging level. See {external+python:ref}`Logging Levels<levels>` for a list of levels.
+
+```python
+log_level = 'INFO'
+```
+
+In the `default_hooks.logger` field, you can specify the logging interval during training and testing. And all
+available arguments can be found in the [LoggerHook docs](mmengine.hooks.LoggerHook).
+
+```python
+default_hooks = [
+    ...
+    # print log every 100 iterations.
+    logger=dict(type='LoggerHook', interval=100),
+    ...
+]
+```
+
+In the `log_processor` field, you can specify the log smooth method. Usually, we use a window with length of 10
+to smooth the log and output the mean value of all information. If you want to specify the smooth method of
+some information finely, see the [LogProcessor docs](mmengine.runner.LogProcessor).
+
+In the `visualizer` field, you can specify multiple backends to save the log information, such as TensorBoard
+and WandB. More details can be found in the [Visualizer section](#visualizer).
+
+## Custom Hooks
+
+The hook mechanism is widely used in all OpenMMLab libraries. Through hooks, you can plug in many
+functionalities without modifying the source code of the runner.
```

Comment:
some functionalities above involve using hooks, it might be better to introduce `hook` in the beginning of file, or at least  re-mention the `CheckpointHook` and `LogHook` in here somehow